int32[64] data
int8 slave_id
int8 func_code

